docs/api/vast-parser.md

```diff
@@ -45,17 +45,37 @@ const vastParser = new VASTParser();
 ```
 
 ## Events<a name="events"></a>
-`VASTParser` extends [`EventEmitter`](https://nodejs.org/api/events.html#events_class_eventemitter), therefore is possible to add event listeners like this:
+`VASTParser` extends [`EventEmitter`](https://nodejs.org/api/events.html#events_class_eventemitter), therefore is possible to add event listeners.
+Here is the list of event emitted by the class:
+### VAST-error
+
+Event listener that provides an object with error information such as error code, error message, extensions and system.
```

Comment:
To be 'strict' here we're documenting an event and not an event listener ;)

We can rephrase it to something like:

The event is triggered whenever the parser "bla bla bla (explain when the event should be fired)" and carries the following data:
- `ERRORCODE [optional]` - bla bla bla

Same for the other events